psutil/arch/windows/process_handles.c

```diff
@@ -82,8 +82,7 @@ psutil_enum_handles(PSYSTEM_HANDLE_INFORMATION_EX *handles) {
 
 
 static int
-psutil_get_filename(LPVOID lpvParam) {
-    HANDLE hFile = *((HANDLE*)lpvParam);
```

Comment:
You forgot to revert this (no changes should be made to this file) =)

Reply:
done